docker-compose.yml

```diff
@@ -0,0 +1,53 @@
+version: '3.8'
+services:
+
+  postgres:
+    image: postgres:alpine
+    volumes:
+      - pgdata:/var/lib/postgresql/data
+    environment:
+        POSTGRES_PASSWORD: showtime
+        POSTGRES_USER: showtime
+        POSTGRES_DB: showtime
+
+  brave:
+    image: registry.comp.ystv.co.uk/brave:latest
```

Comment:
To help people who don't have access to the registry, we could put a reference to the repo where they can build the image themselves? For this it's [github.com/ystv/brave](github.com/ystv/brave)